### ISSUE TITLE
why "loss" is so big when training Qwen3-30B-A3B

### ISSUE THREAD

**Issue by Cass998** [QwenLM/Qwen3-VL#2093]
环境：
8*A800(80G)
torch 2.5.1
transformer_engine 2.15.0
megatron-core 0.12.1

训练脚本：

NPROC_PER_NODE=8
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \
torchrun \
  --nproc_per_node=$NPROC_PER_NODE \
  --master_addr=127.0.0.1 \
  --master_port=29500 \
/root/miniconda3/envs/xxx/bin/megatron sft \
    --model ${HF_MODEL_PATH} \
    --save_safetensors true \
    --dataset ${DATASET_PATH} \
    --output_dir ${OUTPUT_DIR} \
    --tuner_type lora \
    --lora_rank 8 \
    --lora_alpha 32 \
    --target_modules all-linear \
    --split_dataset_ratio 0.01 \
    --finetune true \
    --dataset_num_proc 8 \
    --dataloader_num_workers 8 \
    --num_train_epochs 1 \
    --max_length 2048 \
    --micro_batch_size 1 \
    --global_batch_size 4 \
    --recompute_granularity full \
    --recompute_method uniform \
    --recompute_num_layers 1 \
    --lr 1e-5 \
    --lr_warmup_fraction 0.05 \
    --min_lr 1e-6 \
    --tensor_model_parallel_size 4 \
    --sequence_parallel true \
    --expert_model_parallel_size 8 \
    --moe_permute_fusion true \
    --moe_grouped_gemm true \
    --moe_shared_expert_overlap true \
    --moe_expert_capacity_factor 2 \
    --eval_steps 500 \
    --save_steps 500 \
    --load_from_cache_file true \
    --cross_entropy_loss_fusion true \
    --no_save_optim true \
    --no_save_rng true \
    --packing true \
    --attention_backend flash

训练结果：

<img width="2219" height="310" alt="Image" src="https://github.com/user-attachments/assets/60995f46-5bb1-44b1-8c3c-bcab26c515b6" />

在npu（8张a3的昇腾卡）就没有这个问题，loss在1.4浮动

**Comment by JinghuaGao** [comment]
Qwen3-30B-A3B 是VL 模型吗？ 当前qwen 的VL 系列 Omni 系列， 以及 不带标志的 类似于 qwen3.6-35B-A3B 系列。 傻傻搞不清楚，似乎都支持 图片，视频的输入。

**Comment by JJJYmmm** [comment]
@Cass998  LLM的问题可以report到 https://github.com/nvidia/megatron-lm or https://github.com/QwenLM/Qwen3 🫡

**Comment by JJJYmmm** [comment]
@JinghuaGao Qwen3不支持，Qwen3-VL/Qwen3-Omni支持图片/视频，Qwen3.5/3.6开始原生支持图片/视频。
